tests/subsys/fs/multi-fs/src/main.c

```diff
@@ -39,28 +40,36 @@ static void test_teardown(void)
 void test_main(void)
 {
 	ztest_test_suite(multifs_fs_test,
-		ztest_unit_test(test_fat_mount),
-		ztest_unit_test_setup_teardown(test_nffs_mount,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_mkdir),
-		ztest_unit_test_setup_teardown(test_nffs_mkdir,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_readdir),
-		ztest_unit_test(test_fat_rmdir),
-		ztest_unit_test_setup_teardown(test_nffs_readdir,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_open),
-		ztest_unit_test_setup_teardown(test_nffs_open,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_write),
-		ztest_unit_test_setup_teardown(test_nffs_write,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_read),
-		ztest_unit_test(test_fat_close),
-		ztest_unit_test_setup_teardown(test_nffs_read,
-					       test_setup, test_teardown),
-		ztest_unit_test(test_fat_unlink),
-		ztest_unit_test_setup_teardown(test_nffs_unlink,
-					       test_setup, test_teardown));
+#ifdef CONFIG_FILE_SYSTEM_SHELL
```

Comment:
this is not the right way to do it, look at doc/subsystems/test/ztest.rst

Reply:
@nashif 
ok, i checked the documentation...
just verifying, so that i am on same page ..

Using  conditions #ifdef  in test function instead of ztest calls, as shared below..?
#ifdef CONFIG_FILE_SYSTEM_SHELL
void test_mount_shell(void)
{
        zassert_true(1, "true");
}
#else
void test_mount_shell(void)
{
        ztest_test_skip(); // for tests not using shell mount
}
#endif

Reply:
@nashif : done, pls check